src/components/common/SEO/SEO.tsx

```diff
@@ -0,0 +1,79 @@
+import Head from 'next/head'
+import { FC, ReactNode } from 'react'
+
+import config from '../../../config/seo_meta.json'
+import { OpenGraphSEO } from './OpenGraphSEO'
+import { TwitterCardType, TwitterSEO, TwitterSEOProps } from './TwitterSEO'
+
+interface Props {
+  url?: string
+  title?: string
+  description?: string
+  twitter?: TwitterSEOProps
+  robots?: string
+  children?: ReactNode
+}
+
+export const SEO: FC<Props> = ({
+  url,
+  title,
+  description,
+  twitter,
+  robots,
+  children,
+}) => {
+  const formattedTitle = title
+    ? `${config.titleTemplate.replace(/%s/g, title)}`
+    : config.title
+  return (
+    <>
+      <Head>
+        <title key="title">{formattedTitle}</title>
+        <meta
+          key="description"
+          name="description"
+          content={description ?? config.description}
+        />
+        <meta key="robots" name="robots" content={robots ?? 'index,follow'} />
+        <meta
+          key="googlebot"
+          name="googlebot"
+          content={robots ?? 'index,follow'}
+        ></meta>
+        <meta
+          name="google-site-verification"
+          content="0Jp7zERBL5i76DiM-bODvBGgbjuVMEQGSuwOchP_ZnE"
+        />
+      </Head>
+
+      <TwitterSEO
```

Comment:
**suggestion** i guess it's not really search engine optimizaiton here. Maybe:

```suggestion
      <TwitterMetaTags
```

Reply:
Yep definitely agree - have updated TwitterSEO and OpenGraphSEO to reflect this